css/style.css

```diff
@@ -176,144 +180,182 @@ body {
 
 /* Mobile */
 @media screen and (max-width: 480px) {
+  * {
+    box-sizing: border-box;
+  }
+
   body {
     background-color: var(--white);
     font-family: 'Inter', sans-serif;
     margin: 0;
     padding: 0;
   }
 
-  .menu {
+  .x-icon {
     display: none;
   }
 
-  /* container */
-  .container {
+  .menu {
     display: flex;
     flex-direction: column;
-    justify-content: flex-start;
-    align-items: center;
-    height: 667px;
-    width: 375px;
+    align-items: flex-start !important;
+    width: 100%;
+    padding-top: 2rem;
+    position: absolute;
+    opacity: 100%;
+    background-color: white;
   }
+}
 
-  /* nav-bar */
-  .nav-bar {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-items: center;
-    width: 375px;
-    height: 48px;
-    padding-top: 44px;
-    padding-bottom: 20px;
-  }
+.menu ul {
+  width: 100%;
+}
 
-  .welcome {
-    font-weight: 600;
-    font-size: 20px;
-    font-style: normal;
-    line-height: 28px;
-    width: 90px;
-    height: 28px;
-    padding-left: 12px;
-    color: var(--green-cayn);
-  }
+.menu ul li {
+  padding: 1rem;
+  padding-top: 2rem;
+  width: 100%;
+}
 
-  .ham-menu {
-    display: flex;
-    align-items: flex-start;
-    padding: 8px;
-    padding-right: 24px;
-  }
+.menu ul li a {
+  text-decoration: none;
+  color: #3a4a42;
+  height: 24px;
+  font-weight: 600;
+  font-size: 32px;
+  margin: 2rem;
+  width: 100%;
+}
 
-  /* intro-section */
-  .background {
-    background-image: url(../images/header-illsutration-mobile.svg);
-    background-repeat: no-repeat;
-    background-size: cover;
-    width: 100%;
-    height: auto;
-  }
+.navbar-links {
+  display: none;
+}
 
-  .intro {
-    width: 327px;
-    height: 192px;
-    line-height: 130%;
-    text-align: center;
-    padding-left: 24px;
-    margin-top: 64px;
-  }
+.navbar-links.active {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  position: relative;
+}
 
-  .primary-text {
-    width: 100%;
-    font-style: normal;
-    font-size: 40px;
-    line-height: 52px;
-    font-weight: 700;
-    margin: 0;
-    padding-bottom: 0;
-  }
+/* container */
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  height: auto;
+  padding-top: 1rem;
+  width: 100%;
+  z-index: 1;
+}
 
-  .primary {
-    color: var(--cayn-blue);
-  }
+/* nav-bar */
+.welcome {
+  font-weight: 600;
+  font-size: 20px;
+  font-style: normal;
+  line-height: 28px;
+  width: 90px;
+  height: 28px;
+  padding-left: 12px;
+  color: var(--green-cayn);
+}
 
-  /* span */
-  .career {
-    color: var(--green);
-  }
+.ham-menu {
+  padding-right: 1rem;
+  padding-left: 320px;
+  display: flex;
+  align-items: flex-start;
+  position: absolute;
+}
```

Comment:
- [ ] The hamburger button and X should be removed on desktop, kindly update it with usage of media queries 
![image](https://user-images.githubusercontent.com/62705947/202145714-4ef0f088-4b56-40df-929c-5b1f123a85ba.png)

Reply:
Thank you for taking the time to review my code :+1: 
I have implemented this change.